src/main/java/com/regnosys/rosetta/common/transform/TransformType.java

```diff
@@ -23,11 +23,11 @@
 import java.util.function.Function;
 
 public enum TransformType {
-    PRE_TRANSLATE("translate/pre"),
-    TRANSLATE("translate", null),
-    TRANSLATE_1_5("ingestions", null),
-    POST_TRANSLATE("translate/post"),
-
+    PRE_TRANSLATE("ingest/pre"),
+    TRANSLATE("ingest", null),
```

Comment:
The previously "translate" path was unused.  I think the preferred path (that will be used in each repo resources folder) is "ingest"